modules/sdk-coin-tia/src/lib/utils.ts

```diff
@@ -0,0 +1,237 @@
+import { Coin } from '@cosmjs/stargate';
+import { InvalidTransactionError, TransactionType } from '@bitgo/sdk-core';
+
+import BigNumber from 'bignumber.js';
+import { TxRaw } from 'cosmjs-types/cosmos/tx/v1beta1/tx';
+import * as crypto from 'crypto';
+
+import * as constants from './constants';
+import {
+  CosmosLikeTransaction,
+  DelegateOrUndelegeteMessage,
+  FeeData,
+  MessageData,
+  SendMessage,
+  WithdrawDelegatorRewardsMessage,
+  CosmosConstants,
+  CosmosLikeUtils,
+} from '@bitgo/abstract-cosmos';
+
+export class TiaUtils extends CosmosLikeUtils {
+  /**
+   * Validates if the address matches with regex @see accountAddressRegex
+   *
+   * @param {string} address
+   * @returns {boolean} - the validation result
+   */
+  isValidAddress(address: string): boolean {
+    return constants.accountAddressRegex.test(address);
+  }
+
+  /**
+   * Validates if the address matches with regex @see accountAddressRegex
+   *
+   * @param {string} address
+   * @returns {boolean} - the validation result
+   */
+  isValidValidatorAddress(address: string): boolean {
+    return constants.validatorAddressRegex.test(address);
+  }
+
+  validateWithdrawRewardsMessage(withdrawRewardsMessage: WithdrawDelegatorRewardsMessage) {
```

Comment:
for this and all the methods in this class which perform basic validations common across chains, they could be put into a common parent utility class